UPGRADE.md

```diff
@@ -1,5 +1,21 @@
+# Upgrade to 3.x General Notes
+
+We recommend you upgrade to DBAL 3 first before upgrading to ORM 3. See
+the DBAL upgrade docs: https://github.com/doctrine/dbal/blob/3.10.x/UPGRADE.md
+
+It is not required to upgrade to DBAL 4 or up yet.
```

Comment:
I know what you mean, but I think it could be misinterpreted. People might not get that the alternative to what you are proposing isn't to upgrade to ORM 3 then to DBAL 3, which is impossible, but to do both upgrades at once.

 I'd recommend a more explicit wording:

```
Rather than doing several major upgrades at once, we recommend you do the following:

- upgrade to DBAL 3
- deploy and monitor
- upgrade to ORM 3
- deploy and monitor
- upgrade to DBAL 4
- deploy and monitor
```